utils/build.ps1

```diff
@@ -2774,7 +2774,9 @@ if (-not $SkipBuild) {
 
   $InstallTo = "$($HostArch.ToolchainInstallRoot)\usr"
   foreach ($Arch in $WindowsSDKArchs) {
-    Invoke-BuildStep Build-Sanitizers Windows $Arch $InstallTo
+    if ($Arch.ShortName = "x64") {
```

Comment:
Can the sanitizers work on x86 as well?

Reply:
Yes, but we didn't test it and since the arm64 test was negative, it seemed reasonable to stick to what we know works.